packages/namekit-react/src/components/CurrencySymbol.tsx

```diff
@@ -0,0 +1,161 @@
+import { PriceCurrencyFormat, Currency } from "@namehash/ens-utils";
+import { Tooltip } from "./Tooltip";
+
+import { UsdSymbol } from "./icons/UsdSymbol";
+import { UsdcSymbol } from "./icons/UsdcSymbol";
+import { WethSymbol } from "./icons/WethSymbol";
+import { EthSymbol } from "./icons/EthSymbol";
+import { DaiSymbol } from "./icons/DaiSymbol";
+import React from "react";
+
+/**
+ * The size of the `CurrencySymbol`.
+ * This is defined as a mapping from a `CurrencySymbolSize` to a number.
+ * This number represents the size (width and height) a `CurrencySymbol` should be displayed at, in pixels.
+ */
+export const CurrencySymbolSize = {
```

Comment:
Shouldn't we rename this to `CurrencyIconSize` to be more clear it is only relevant for icons and not other symbologies?